src/main/java/com/roommatematcher/enterprise/DAO/IProfileDAO.java

```diff
@@ -0,0 +1,8 @@
+package com.roommatematcher.enterprise.DAO;
+
+public interface IProfileDAO {
```

Comment:
What is the purpose of this interface?  Without methods or JavaDoc, I'm not certain.
It looks like incomplete functionality.